README.md

```diff
@@ -129,103 +201,166 @@ module.exports = {
     // ...
   ],
   // ...
-}
+};
 ```
 
 #### Multiple parsers
 
 If you want to switch the parser for each lang, specify the object.
 
+For example in `eslint.config.js`:
+
+```js
+import tsParser from "@typescript-eslint/parser";
+import espree from "espree";
+export default [
+  {
+    files: ["**/*.svelte", "*.svelte"],
+    languageOptions: {
+      parser: svelteParser,
+      parserOptions: {
+        parser: {
+          ts: tsParser,
+          js: espree,
+          typescript: tsParser,
+        },
+      },
+    },
+  },
+];
+```
+
+For example in `.eslintrc.*`:
+
 ```json
 {
-    "parser": "svelte-eslint-parser",
-    "parserOptions": {
-        "parser": {
-            "ts": "@typescript-eslint/parser",
-            "js": "espree",
-            "typescript": "@typescript-eslint/parser"
-        }
+  "parser": "svelte-eslint-parser",
+  "parserOptions": {
+    "parser": {
+      "ts": "@typescript-eslint/parser",
+      "js": "espree",
+      "typescript": "@typescript-eslint/parser"
     }
+  }
 }
 ```
 
-#### Parser Object
```

Comment:
[ask]

Can we remove this section?

Reply:
Since the explanation of `eslint.config.js` uses the parser object, there is no need to explain it again, so I removed it. I added that feature for support for `eslint.config.js`.